tests/unit/test_charm.py

```diff
@@ -219,16 +248,13 @@ def test_given_statefulset_is_not_patched_when_config_changed_then_statefulset_i
             )
         )
         self.mock_lightkube_client_get.return_value = test_statefulset
-        self.mock_check_output.return_value = b"1.1.1.1"
-        self.harness.add_storage("config", attach=True)
-        self.set_f1_relation_data()
 
         self.harness.update_config(key_values={})
         self.harness.evaluate_status()
```

Comment:
I think that we do not need the to call `evaluate_status` on this test and on those that does not test the status of the charm.